### ISSUE TITLE
[BUG] macOS have some errors

### ISSUE THREAD

**Issue by zer0yu** [cyrus-and/fracker#7]
macOS

Env
```
❯ php -version
PHP 7.3.11 (cli) (built: Jun  5 2020 23:50:40) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.11, Copyright (c) 1998-2018 Zend Technologies
    with Xdebug v2.7.0, Copyright (c) 2002-2019, by Derick Rethans
```

```
~/Downloads/fracker/ext master                                                             Py anaconda2 19:36:08
❯ ~/php-private/phpize
Configuring for:
PHP Api Version:         20180731
Zend Module Api No:      20180731
Zend Extension Api No:   320180731
```

Error after i exec `make`
```
/Users/zeroyu/Downloads/fracker/ext/src/tracing/trace_fracker.c:91:20: error: use of undeclared identifier
      'SOL_TCP'
    setsockopt(fd, SOL_TCP, TCP_CORK, &cork, sizeof(cork));
                   ^
/Users/zeroyu/Downloads/fracker/ext/src/tracing/trace_fracker.c:91:29: error: use of undeclared identifier
      'TCP_CORK'
    setsockopt(fd, SOL_TCP, TCP_CORK, &cork, sizeof(cork));
                            ^
/Users/zeroyu/Downloads/fracker/ext/src/tracing/trace_fracker.c:101:20: error: use of undeclared identifier
      'SOL_TCP'
    setsockopt(fd, SOL_TCP, TCP_CORK, &cork, sizeof(cork));
                   ^
/Users/zeroyu/Downloads/fracker/ext/src/tracing/trace_fracker.c:101:29: error: use of undeclared identifier
      'TCP_CORK'
    setsockopt(fd, SOL_TCP, TCP_CORK, &cork, sizeof(cork));
                            ^
4 errors generated.
make: *** [src/tracing/trace_fracker.lo] Error 1
```

**Comment by cyrus-and** [comment]
The macOS support is not really a goal... but if the fix is trivial I could include it.

In the meantime try to comment out the two `setsockopt` invocations in the following and see it succeeds:

https://github.com/cyrus-and/fracker/blob/62d599ceef6135eab1d5bd6afe9dc8123dea8ba8/ext/src/tracing/trace_fracker.c#L91-L101

**Comment by zer0yu** [comment]
But is this `setsockopt` function for sending PHP stack information?

**Comment by cyrus-and** [comment]
It's just there for performance tuning.

**Comment by zer0yu** [comment]
thx, i will try

**Comment by cyrus-and** [comment]
It should be fixed in feafb7cc742b08ab5994ee32e28d9249724030b7, please give it a try.

**Comment by zer0yu** [comment]
it works. thx
